iac-extensions/arm/src/test/java/org/sonar/iac/arm/tree/impl/json/ResourceDeclarationImplTest.java

```diff
@@ -87,16 +89,14 @@ void shouldParseResource() {
     assertThat((ArmTree) children.get(2)).is(STRING_LITERAL).has("value", "Microsoft.Kusto/clusters").hasRange(4, 14, 4, 40);
   }
 
-  @ParameterizedTest
-  @ValueSource(strings = {
-    "\"type\": 5,                            \"apiVersion\": \"2022-12-29\", \"name\": \"myResource\"",
-    "\"type\": \"Microsoft.Kusto/clusters\", \"apiVersion\": \"2022-12-29\", \"name\": 5             ",
-  })
-  void shouldFailOnInvalidPropertyValueType(String invalidPropertyType) {
+  @Test
+  void shouldFailOnInvalidPropertyValueType() {
     String code = code("{",
```

Comment:
Please replace `code()` with Text Block. If possible please replace in the whole file.